tools/topology/m4/tdfb_defines.m4

```diff
@@ -0,0 +1,6 @@
+define(DEF_TDFB_PRIV, concat(`tdfb_priv_', PIPELINE_ID))
+define(DEF_TDFB_BYTES, concat(`tdfb_bytes_', PIPELINE_ID))
+define(DEF_TDFB_ONOFF_ENUM, concat(`tdfb_enable_', PIPELINE_ID))
```

Comment:
The control when listed by amixer controls would look like name='TDFB10.0 tdfb_enable_10'. I think a single word here would be sufficient (to remain unique) to make it like name='TDFB10.0 enable'. But is "enable" intuitive. The enum values are currently "beam on" (0 - default) and "beam off (1)".

Reply:
These are now commented with dnl in m4 to not add the controls not supported by current kernel.